cloud-sql/mysql/Program.cs

```diff
@@ -32,6 +34,10 @@ public class Program
         public static void Main(string[] args)
         {
             BuildWebHost(args).Build().Run();
+            // Create Database table if it does not exist.
+            var connectionString = new MySqlConnection().GetMySqlConnectionString();
+            DbConnection connection = new MySqlConnection(connectionString.ConnectionString);
+            connection.InitializeDatabase();

```

Comment:
Becuase the proposed changes to Polly, we now need something like...
```suggestion
            StartupExtensions.InitializeDatabase();

```

Reply:
Done.